spec/models/game_spec.rb

```diff
@@ -8,5 +8,27 @@
 			expect(g.pieces.count).to eq 32
 		end
 	end
+
+	describe "checking the status of game if in check" do
+
+		it "should return true if the game is in check" do
+			@game = Game.new
+			@white_king = King.create(
+				x_coordinate: 2,
+				y_coordinate: 2,
+				game: @game,
+				color: "white",
+				)
+
+			@black_rook = Rook.create(
+				x_coordinate: 2,
+				y_coordinate: 4,
+				game: @game,
+				color: "white",
+				)
+			expect(@black_rook.valid_move?(2, 2)).to eq true
+
+		end
+	end
 end
 

```

Comment:
This test is irrelevant because it's checking for a valid move. We want to see if the game is in check by using the is_check method